src/modules/get-graphql-data-from-files.ts

```diff
@@ -0,0 +1,85 @@
+import fs from 'fs'
+import 'graphql-import-node'
+import { DocumentNode } from 'graphql'
+import { IResolvers } from 'graphql-tools'
+import { mergeTypeDefs } from '@graphql-toolkit/schema-merging'
+
+export function requireAll<T>(dir: string): T[]
+export function requireAll<T>(
+  dir: string,
+  filter: (fileName: string) => boolean
+): T[]
+export function requireAll<T>(
+  dir: string,
+  filter: (fileName: string) => boolean = () => {
+    return true
+  }
+): T[] {
+  const modules = []
+  const files = fs.readdirSync(dir)
+  for (const fileName of files) {
+    const filePath = dir + '/' + fileName
+    if (fs.statSync(filePath).isDirectory()) {
+      modules.push(...requireAll<T>(filePath, filter))
+    } else {
+      if (filter(fileName)) {
+        modules.push(require(filePath))
+      }
+    }
+  }
+  return modules
+}
+export function getGraphqlsFromFile(dir: string): DocumentNode
+export function getGraphqlsFromFile(
+  dir: string,
+  filter: (fileName: string) => boolean
+): DocumentNode
+export function getGraphqlsFromFile(
+  dir: string,
+  filter: (fileName: string) => boolean = () => {
+    return true
+  }
+): DocumentNode {
+  const results = requireAll<DocumentNode>(dir, (fileName: string) => {
+    const splitedFileName = fileName.split('.')
+    if (
+      splitedFileName[splitedFileName.length - 1] === 'graphql' &&
```

Comment:
In Node.js' `path` module, there is a method called `extname()` where it parses the extension name from a file path.
It will help you write much simpler and robust code.

```ts
import path from 'path'

const extName = path.extname('schema.graphql')

console.log(extName) // 'graphql'
```